packages/react/src/Trans.tsx

```diff
@@ -27,8 +27,8 @@ export function Trans(props: TransProps): React.ReactElement<any, any> | null {
   const { i18n, defaultComponent } = useLingui()
   const { render, component, id, message, formats } = props
 
-  const values = { ...props.values }
-  const components = { ...props.components }
+  const values = { ...(props.values || {}) }
+  const components = { ...(props.components || {}) }
 
   if (values) {
```

Comment:
noticed that `values` will always be truthy here

Reply:
Nice catch, but looks like changes from this PR did not affect this.